javascript_nodejs/1.console-echobot/README.md

```diff
@@ -0,0 +1,51 @@
+This sample shows how to create a simple Echo bot that you can talk to from the terminal window..
+
+# To try this sample
+- Clone the repository
+    ```bash
+    git clone https://github.com/Microsoft/botbuilder-samples.git
+    ```
+- In a terminal, navigate to javascript_nodejs/1.console-echobot
+    ```bash
+    cd javascript_nodejs/1.console-echobot
+    ```
+    - Point to the myget feed 
+    ```bash
+    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    ```
+- Install modules and start the bot
+    ```bash
+    npm i & npm start
+    ```
+- To reset registry, you can do
+    ```bash
+    npm config set registry https://registry.npmjs.org/
+    ```
+
+# Testing the bot 
+
+After running `npm start`, the bot will present a prompt directly in the terminal window.
+
+Send messages to the bot by typing them into the terminal.  The bot will echo your message back to you!
+
+# Adapters
+
+Adapters such as the `ConsoleAdapter` provide an abstraction for your bot to work with a variety of environments and messaging platforms.
+
+The adapter is responsible for directing incoming and outgoing communication, authentication, and so on. Adapters for different platforms and
+messaging technologies differ internally, but achieve the same goal - connecting bots to a stream of messages and events from users. 
+When your bot receives an activity, the adapter wraps up everything about that activity, creates a [context object](https://github.com/Microsoft/botbuilder-js/blob/master/doc/botbuilder/classes/botbuilder.turncontext.md), passes it to your bot's application logic,
+and sends responses generated by your bot back to the user's channel.
+
+In most situations, we don't work with the adapter directly, but it's good to know it's there and what it does. In this example, we're using the
+`ConsoleAdapter`, which provides a very simple way to exchange messages - they're sent and received via the terminal window. Using the ConsoleAdapter
+is a great way of getting started quickly with Botbuilder. It also allows you to build and test bots that operate on your local machine without any
```

Comment:
BotBuilder vs Botbuilder.

Reply:
"It also" isn't a good way to start a sentence. Consider simplifying the grammar.